services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -1312,8 +1312,11 @@ private Integer fetchSystemStoreSchemaId(String clusterName, String storeName, S
       }
       return valueSchemaId;
     }
-    ControllerClient controllerClient = ControllerClient
-        .constructClusterControllerClient(clusterName, getLeaderController(clusterName).getUrl(false), sslFactory);
+
+    ControllerClient controllerClient = ControllerClient.constructClusterControllerClient(
+        clusterName,
+        getLeaderController(clusterName).getUrl(getSslFactory().isPresent()),
```

Comment:
What do you think if we add a final member variable in VeniceHelixAdmin, like isSslEnabled?